examples/user_server_test.rs

```diff
@@ -180,15 +180,15 @@ pub async fn delete_user() {
     CONTEXT
         .run(&request)
```

Comment:
could we try to use each new request at least once (so we can cover them a bit on "real life" tests)

Reply:
> use each new request at least once

wdym? the request here (l. 178)
```rs
let request = Request::delete(path!["users", id]);
```
is used twice (l. 181 & l. 189)

Reply:
I said request, I meant methods sorry

Reply:
ohh yes, of course, I'll add test cases for those!

Reply:
@cbonaudo see [aa8f07a](https://github.com/iomentum/restest/pull/28/commits/aa8f07a2eb377d8e332e7c3951c7dfbde07b4281)